extensions/MyDashboard/lib/Queries.pm

```diff
@@ -313,6 +313,9 @@ sub query_flags {
     # Skip this flag if the bug is not visible to the user
     next if !$visible_bugs{$flag->{'bug_id'}};
 
+    # Skip this flag unless it's specifically requestable and has a requestee
+    next unless $flag->{'requestee'};
+
```

Comment:
I would instead filter out the flags without requestee in the DB instead of looping over 700+ flags here. In lines 256-261:

  if ($type eq 'requestee') {
    $match_params->{'requestee_id'} = $user->id;
  }
  else {
    $match_params->{'setter_id'} = $user->id;
    $match_params->{'requestee_id'} = NOT_NULL;
  }

Reply:
Thanks! Didn’t know `NOT_NULL` was available. Fixing now.